docker/containers-data/suricata/etc/selks6-addin.yaml

```diff
@@ -383,7 +383,7 @@ outputs:
         - nfs
         - smb
         - tftp
-        - ikev2
+        - ike
```

Comment:
This should be in a separate commit as it has no relation to the reaply/clean up scripts.

Reply:
you're right, done.